packages/metro-config/package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "@expo/config": "3.3.31-alpha.0",
     "chalk": "^4.1.0",
-    "getenv": "^0.7.0",
```

Comment:
please update everywhere and run yarn.

Reply:
updated all instances of this dependency and run yarn. quick question though, should I also commit the yarn.lock file?

Reply:
yes

Reply:
done

Reply:
something broke the CI tests (run `yarn start` in packages (should be pretty quick)).

Reply:
yes, something actually broke. I'm attaching the output of `yarn start` while I investigate.
```
❯ yarn start
yarn run v1.22.10
$ lerna --ignore "@expo/{dev-tools,next-adapter,electron-adapter}" --ignore expo-optimize --ignore pod-install --ignore uri-scheme run watch --parallel
Configuring yargs through package.json is deprecated and will be removed in a future major release, please use the JS API instead.
Configuring yargs through package.json is deprecated and will be removed in a future major release, please use the JS API instead.
lerna notice cli v3.4.1
lerna WARN project Deprecated key "commands" found in lerna.json
lerna WARN project Please update "commands" => "command"
lerna info versioning independent
lerna notice filter excluding ["@expo/{dev-tools,next-adapter,electron-adapter}","expo-optimize","pod-install","uri-scheme"]
lerna info filter [
lerna info filter   '!@expo/{dev-tools,next-adapter,electron-adapter}',
lerna info filter   '!expo-optimize',
lerna info filter   '!pod-install',
lerna info filter   '!uri-scheme'
lerna info filter ]
lerna info Executing command in 17 packages: "yarn run watch"
(node:61731) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 2 error listeners added to [WriteStream]. Use emitter.setMaxListeners() to increase limit
@expo/config-types: $ tsc --watch
@expo/config: $ tsc --watch
@expo/plist: $ tsc --watch
@expo/dev-server: $ tsc --watch
@expo/config-plugins: $ tsc --watch
expo-pwa: $ tsc --watch
@expo/schemer: $ tsc --watch
@expo/xdl: $ concurrently "tsc --emitDeclarationOnly --watch" "gulp watch"
@expo/webpack-config: $ tsc --watch
@expo/image-utils: $ tsc --watch
expo-codemod: $ tsc --watch
@expo/metro-config: $ tsc --watch
expo-cli: $ tsc --watch
@expo/json-file: $ tsc --watch
@expo/pkcs12: $ tsc --watch
@expo/package-manager: $ tsc --watch
@expo/osascript: $ tsc --watch
@expo/config-types:
2:49:15 PM - Starting compilation in watch mode...
@expo/config:
2:49:15 PM - Starting compilation in watch mode...
@expo/config-plugins:
2:49:15 PM - Starting compilation in watch mode...
@expo/dev-server:
2:49:15 PM - Starting compilation in watch mode...
@expo/plist:
2:49:15 PM - Starting compilation in watch mode...
expo-pwa:
2:49:15 PM - Starting compilation in watch mode...
@expo/schemer:
2:49:15 PM - Starting compilation in watch mode...
@expo/webpack-config:
2:49:16 PM - Starting compilation in watch mode...
@expo/image-utils:
2:49:16 PM - Starting compilation in watch mode...
expo-codemod:
2:49:16 PM - Starting compilation in watch mode...
@expo/metro-config:
2:49:16 PM - Starting compilation in watch mode...
expo-cli:
2:49:16 PM - Starting compilation in watch mode...
@expo/json-file:
2:49:16 PM - Starting compilation in watch mode...
@expo/xdl: [0]
2:49:16 PM - Starting compilation in watch mode...
@expo/xdl: [0]
@expo/pkcs12:
2:49:16 PM - Starting compilation in watch mode...
@expo/package-manager:
2:49:16 PM - Starting compilation in watch mode...
@expo/osascript:
2:49:16 PM - Starting compilation in watch mode...
@expo/xdl: [1] [14:49:17] Using gulpfile ~/workspace/expo-cli/packages/xdl/gulpfile.js
@expo/xdl: [1] [14:49:17] Starting 'watch'...
@expo/xdl: [1] [14:49:17] Starting 'babel'...
@expo/config-types: 2:49:20 PM - Found 0 errors. Watching for file changes.
@expo/config: 2:49:21 PM - Found 0 errors. Watching for file changes.
@expo/plist: 2:49:22 PM - Found 0 errors. Watching for file changes.
@expo/schemer: 2:49:22 PM - Found 0 errors. Watching for file changes.
@expo/config-plugins: 2:49:23 PM - Found 0 errors. Watching for file changes.
expo-codemod: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/json-file: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/dev-server: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/pkcs12: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/image-utils: src/Image.ts(129,25): error TS2339: Property 'getType' does not exist on type 'typeof import("/Users/andressb/workspace/expo-cli/node_modules/@types/mime/index")'.
@expo/image-utils: src/Image.ts(139,63): error TS2339: Property 'getExtension' does not exist on type 'typeof import("/Users/andressb/workspace/expo-cli/node_modules/@types/mime/index")'.
@expo/image-utils: 2:49:24 PM - Found 2 errors. Watching for file changes.
@expo/osascript: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/metro-config: 2:49:24 PM - Found 0 errors. Watching for file changes.
expo-pwa: src/index.ts(213,13): error TS2783: 'backgroundColor' is specified more than once, so this usage will be overwritten.
expo-pwa: 2:49:24 PM - Found 1 error. Watching for file changes.
@expo/dev-server:
2:49:24 PM - File change detected. Starting incremental compilation...
@expo/dev-server: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/package-manager: 2:49:24 PM - Found 0 errors. Watching for file changes.
@expo/xdl: [1] [14:49:25] Finished 'babel' after 8.33 s
@expo/xdl: [1] [14:49:25] Starting 'watchBabel'...
@expo/xdl: [1] [14:49:25] Finished 'watchBabel' after 14 ms
@expo/xdl: [1] [14:49:25] Finished 'watch' after 8.35 s
@expo/webpack-config: 2:49:25 PM - Found 0 errors. Watching for file changes.
@expo/webpack-config:
2:49:26 PM - File change detected. Starting incremental compilation...
@expo/webpack-config: 2:49:26 PM - Found 0 errors. Watching for file changes.
@expo/xdl: [0] src/start/resolveNgrok.ts(1,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
@expo/xdl: [0] src/Webpack.ts(3,29): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
@expo/xdl: [0]
@expo/xdl: [0] 2:49:27 PM - Found 2 errors. Watching for file changes.
@expo/xdl: [0]
2:49:27 PM - File change detected. Starting incremental compilation...
@expo/xdl: [0]
@expo/xdl: [0] src/start/resolveNgrok.ts(1,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
@expo/xdl: [0] src/Webpack.ts(3,29): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
@expo/xdl: [0]
@expo/xdl: [0] 2:49:27 PM - Found 2 errors. Watching for file changes.
expo-cli: src/commands/customize.ts(2,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/install.ts(3,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/upgrade.ts(9,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/utils/CreateApp.ts(1,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/utils/CreateApp.ts(132,7): error TS2532: Object is possibly 'undefined'.
expo-cli: src/commands/utils/CreateApp.ts(132,14): error TS2339: Property 'nodeLinker' does not exist on type 'string | object'.
expo-cli:   Property 'nodeLinker' does not exist on type 'string'.
expo-cli: src/commands/utils/typescript/ensureTypeScriptSetup.ts(2,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: 2:49:28 PM - Found 7 errors. Watching for file changes.
expo-cli:
2:49:28 PM - File change detected. Starting incremental compilation...
expo-cli: src/commands/customize.ts(2,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/install.ts(3,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/upgrade.ts(9,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/utils/CreateApp.ts(1,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: src/commands/utils/CreateApp.ts(132,7): error TS2532: Object is possibly 'undefined'.
expo-cli: src/commands/utils/CreateApp.ts(132,14): error TS2339: Property 'nodeLinker' does not exist on type 'string | object'.
expo-cli:   Property 'nodeLinker' does not exist on type 'string'.
expo-cli: src/commands/utils/typescript/ensureTypeScriptSetup.ts(2,33): error TS2307: Cannot find module '@expo/package-manager' or its corresponding type declarations.
expo-cli: 2:49:29 PM - Found 7 errors. Watching for file changes.
```

Reply:
weird thing is if I revert back to the previous commit (that was passing in CI) it also fails now. Any idea what could be happening?

Reply:
7800+ line change in the yarn.lock seems suspicious.

Reply:
absolutely right. started up with a fresh branch and now it works correctly with a minimal change in yarn.lock. I'm going to discard this PR and open a proper one again .